src/components/Tests/Tests.stories.tsx

```diff
@@ -107,4 +107,6 @@ export const Error: Story = {
   }
 };
 
+export const EmptyWithoutSpan: Story = {};
```

Comment:
Seems like there is no difference between `Loading` and `EmptyWithoutSpan` stories